src/main/java/com/jcabi/http/mock/MkAnswerMatchers.java

```diff
@@ -55,6 +55,18 @@ private MkAnswerMatchers() {
     }
 
     /**
+     * Matches the value of {@link MkAnswer#bodyBytes()} against the given
+     * matcher.
+     *
+     * @param matcher The matcher to use.
+     * @return Matcher for checking the body of MkAnswer
+     */
+    public static Matcher<MkAnswer> hasBodyBytes(
```

Comment:
@mkordas If you want to check that your API returned the correct bytes (e.g. an image).